pkg/apis/kubermatic/v1/constraint_template.go

```diff
@@ -41,13 +41,16 @@ type ConstraintTemplate struct {
 	metav1.TypeMeta   `json:",inline"`
 	metav1.ObjectMeta `json:"metadata,omitempty"`
 
+	// Spec specifies the gatekeeper constraint template and kubermatic related spec.
 	Spec ConstraintTemplateSpec `json:"spec,omitempty"`
 }
 
 // ConstraintTemplateSpec is the object representing the gatekeeper constraint template spec and kubermatic related spec.
 type ConstraintTemplateSpec struct {
-	CRD      v1.CRD                     `json:"crd,omitempty"`
-	Targets  []v1.Target                `json:"targets,omitempty"`
+	CRD v1.CRD `json:"crd,omitempty"`
+	Targets []v1.Target `json:"targets,omitempty"`
+
+	// Selector is the object holding the cluster selection filters.
```

Comment:
I would slightly rephrase this - We know this is an object, maybe let's focus on what it's used for. Something like:

```suggestion
	// Selector configures which clusters this constraint template is applied to by providing a label selector.
```